### ISSUE TITLE
Link styling in JupyterLab light theme doesn't have enough contrast to pass WCAG AAA for Normal Text

### ISSUE THREAD

**Issue by andrii-i** [jupyterlab/jupyterlab#14166]
## Description

Link styling in JupyterLab light theme doesn't have enough contrast to pass WCAG AAA for Normal Text

## Reproduce

1. Open [command palette](https://jupyterlab.readthedocs.io/en/stable/user/commands.html) in a notebook
2. Enter "Trust Notebook"
3. Note contrast level of link vs background color in the popup window

WebAim test can also be found [here](https://webaim.org/resources/contrastchecker/?fcolor=1976D2&bcolor=FFFFFF)
![image](https://user-images.githubusercontent.com/26686070/224459732-6cfaa664-f188-432f-ba65-5eb4175bd62b.png)

## Expected behavior

Link styling in JupyterLab light theme does have enough contrast to pass WCAG AAA for Normal Text

## Context

Operating System and version: MacOS 12.6.3
Browser and version: Chrome version 109.0.5414.87 (Official Build) (arm64)
JupyterLab version: head-of-master (WIP 4.0.0)

**Comment by krassowski** [comment]
Currently `--jp-content-link-color` is mapped to `--md-blue-700` in light mode.

Based on WCAG 3.0 draft guidelines using [APCA to the current link colour in light theme](https://www.myndex.com/APCA/?BG=ffffff&TXT=1976d2&DEV=G4g&BUF=A22) we get Lc 71.4 which is below recommended _minimum_ of Lc 75. Bumping it to `--md-blue-800` (`#1565C0`) gives 5.75 contrast against white/Lc 78.2. Although not meeting WCAG 2.0 AAA, it would be rather uncontroversial, incremental change,

**Comment by MMesch** [comment]
The triaging agent has identified this issue as likely resolved. Indeed, analysis of the current codebase shows the `--jp-content-link-color` variable in the light theme now uses `var(--md-blue-900, #0d47a1)` instead of the problematic `--md-blue-700` (#1976d2) mentioned in the issue. This change was implemented through PR #14159 which specifically targeted fixing link colors and explicitly mentions resolving this issue. The darker blue provides significantly better contrast against white backgrounds, addressing the WCAG AAA compliance concern raised.

I am not sure though what your comment and endorsement mean @krassowski and @andrii-i , which appear to have been written after merging that PR. If we are _not_ compliant even after the change to `--md-blue-900`, can you make explicit what remains to be done to resolve this issue? If we _are_ compliant and this issue is resolved, could you close it as completed?

**Comment by krassowski** [comment]
Yes, it passes both AA and AAA in light mode now. I guess I was commenting when looking at a stable version rather than on the dev.
